src/internal/pdf/pdf.go

```diff
@@ -0,0 +1,64 @@
+package pdf
+
+import (
+	"bytes"
+	_ "embed"
+	"errors"
+	"time"
+
+	"github.com/signintech/gopdf"
+)
+
+const (
+	PAGE_WIDTH  = 162
+	PAGE_HEIGHT = 90
+	PAGE_MARGIN = 10
+)
+
+//go:embed fonts/Rubik-Regular.ttf
+var rubikRegular []byte
+
+// Generate a PDF label consisting of the provided `labelText` and the current date
+func GeneratePdf(labelText string) ([]byte, error) {
+	// initialize PDF
+	pdf := gopdf.GoPdf{}
+	pdf.Start(gopdf.Config{PageSize: gopdf.Rect{W: PAGE_WIDTH, H: PAGE_HEIGHT}})
+	pdf.AddPage()
+
+	// use embedded font file in PDF
+	rr := bytes.NewReader(rubikRegular)
+	err := pdf.AddTTFFontByReader("Rubik-Regular", rr)
+	if err != nil {
+		return nil, err
+	}
+
+	err = pdf.SetFont("Rubik-Regular", "", 16)
+	if err != nil {
+		return nil, err
+	}
+
+	// write the label text in the upper-left corner of the document
+	pdf.SetXY(PAGE_MARGIN, 10)
+	err = pdf.SetFont("Rubik-Regular", "", 16)
+	pdf.SetTextColor(0, 0, 0)
+	pdf.Cell(nil, labelText)
+
+	// write the "made on" date information in the lower-left corner of the document
+	pdf.SetXY(PAGE_MARGIN, 55)
+	err = pdf.SetFont("Rubik-Regular", "", 10)
+	pdf.SetTextColor(85, 85, 85)
+	pdf.Cell(nil, "made:")
+
+	pdf.SetXY(PAGE_MARGIN, 67)
+	err = pdf.SetFont("Rubik-Regular", "", 12)
+	pdf.SetTextColor(0, 0, 0)
+	pdf.Cell(nil, time.Now().Local().Format(time.DateOnly))
```

Comment:
Both of these can be shifted down the Y axis by ~3pts; this should match the bottom and left margins